lefthook.yaml

```diff
@@ -42,6 +42,91 @@ pre-commit:
 
+    10.1_check_biome_disable:
+      glob: "**/*.{ts,tsx}"
+      run: |
+        set -e
+        if [ -f .venv/bin/python3 ]; then
+          .venv/bin/python3 .github/workflows/scripts/biome_disable_check.py --files {staged_files}
+        elif [ -f .venv/Scripts/python.exe ]; then
+          .venv/Scripts/python.exe .github/workflows/scripts/biome_disable_check.py --files {staged_files}
+        else
+          echo "Error: Python virtual environment not found."
+          echo "Please run: python3 -m venv .venv && source .venv/bin/activate && pip install -r .github/workflows/requirements.txt"
+          exit 1
+        fi
+
+    10.2_check_code_coverage_disable:
+      glob: "**/*.{ts,tsx}"
+      run: |
+        set -e
+        if [ -f .venv/bin/python3 ]; then
+          .venv/bin/python3 .github/workflows/scripts/code_coverage_disable_check.py --files {staged_files}
+        elif [ -f .venv/Scripts/python.exe ]; then
+          .venv/Scripts/python.exe .github/workflows/scripts/code_coverage_disable_check.py --files {staged_files}
+        else
+          echo "Error: Python virtual environment not found."
+          echo "Please run: python3 -m venv .venv && source .venv/bin/activate && pip install -r .github/workflows/requirements.txt"
+          exit 1
+        fi
+
+    10.3_check_ts_ignore:
+      glob: "**/*.{ts,tsx}"
+      run: |
+        set -e
+        if [ -f .venv/bin/python3 ]; then
+          .venv/bin/python3 .github/workflows/scripts/detect_ts_ignore.py --files {staged_files}
+        elif [ -f .venv/Scripts/python.exe ]; then
+          .venv/Scripts/python.exe .github/workflows/scripts/detect_ts_ignore.py --files {staged_files}
+        else
+          echo "Error: Python virtual environment not found."
+          echo "Please run: python3 -m venv .venv && source .venv/bin/activate && pip install -r .github/workflows/requirements.txt"
+          exit 1
+        fi
+
+    10.4_check_sanitization_disable:
+      glob: "**/*.ts"
+      run: |
+        set -e
+        if [ -f .venv/bin/python3 ]; then
+          .venv/bin/python3 .github/workflows/scripts/check_sanitization_disable.py --files {staged_files}
+        elif [ -f .venv/Scripts/python.exe ]; then
+          .venv/Scripts/python.exe .github/workflows/scripts/check_sanitization_disable.py --files {staged_files}
+        else
+          echo "Error: Python virtual environment not found."
+          echo "Please run: python3 -m venv .venv && source .venv/bin/activate && pip install -r .github/workflows/requirements.txt"
+          exit 1
+        fi
+
+    10.5_check_itskip:
+      glob: "test/**/*.{test,spec}.{ts,tsx}"
+      run: |
+        set -e
+        if [ -f .venv/bin/python3 ]; then
+          .venv/bin/python3 .github/workflows/scripts/itskip_disable_check.py --files {staged_files}
+        elif [ -f .venv/Scripts/python.exe ]; then
+          .venv/Scripts/python.exe .github/workflows/scripts/itskip_disable_check.py --files {staged_files}
```

Comment:
Explain that they may also need to install the python3-venv and/or python-venv packages